.github/workflows/android_arm64.yml

```diff
@@ -21,7 +23,7 @@ jobs:
       skip_workflow: "${{ steps.smart_ci.outputs.skip_workflow }}"
     steps:
       - name: checkout action
-        uses: actions/checkout@v4
+        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
```

Comment:
The dependabot can work with these dependencies: https://github.com/dependabot/dependabot-core/issues/4691.